docs/content/doc/advanced/customizing-gitea.en-us.md

```diff
@@ -63,8 +63,9 @@ the url `http://gitea.domain.tld/assets/image.png`.
 To build a custom logo clone the Gitea source repository, replace `assets/logo.svg` and run
 `make generate-images`. This will update below output files which you can then place in `$GITEA_CUSTOM/public/img` on your server:
```

Comment:
```suggestion
`make generate-images`. `assets/favicon.svg` is a separate copy of `assets/logo.svg` and used for the favicon only. This will update below output files which you can then place in `$GITEA_CUSTOM/public/img` on your server:
```